src/allencell_ml_segmenter/prediction/view.py

```diff
@@ -191,13 +191,16 @@ def showResults(self) -> None:
                     if raw_np_data is not None:
                         self._viewer.add_image(
                             raw_np_data,
-                            f"[raw] {data['raw'].name}",
+                            name=f"[raw] {data['raw'].name}",
+                            metadata={"source_path": str(data["raw"])},
                         )
                     if seg_np_data is not None:
                         self._viewer.add_labels(
                             seg_np_data,
                             name=f"[seg] {data['seg'].name}",
+                            metadata={"source_path": str(data["seg"])},
```

Comment:
I see - `name` and `metadata` are the kwargs?

Do we still need to track this info in the actual Napari layers?  Isnt our thresholdModel dict tracking this info now?

Reply:
Can you make a ticket for simplifying the workflow, and describe code changes, and link it here?